src/test/java/com/hubspot/jinjava/el/ext/JinjavaBeanELResolverTest.java

```diff
@@ -11,10 +15,17 @@ public class JinjavaBeanELResolverTest {
   private JinjavaBeanELResolver jinjavaBeanELResolver;
   private ELContext elContext;
 
+  JinjavaInterpreter interpreter = mock(JinjavaInterpreter.class);
+  JinjavaConfig config = mock(JinjavaConfig.class);
+
   @Before
   public void setUp() throws Exception {
     jinjavaBeanELResolver = new JinjavaBeanELResolver();
     elContext = new JinjavaELContext();
+    elContext
+      .getELResolver()
+      .setValue(elContext, null, ExtendedParser.INTERPRETER, interpreter);
+    when(interpreter.getConfig()).thenReturn(config);
```

Comment:
Could add a couple tests that custom properties/methods are restricted

Reply:
I have added the unit tests to test that custom properties/methods are restricted